best_practices/security.rst

```diff
@@ -29,20 +29,17 @@ site (or maybe nearly *all* sections), use the ``access_control`` area.
 
 .. best-practice::
 
-    Use the ``bcrypt`` encoder for hashing your users' passwords.
+    Use the ``auto`` encoder for hashing your users' passwords.
 
-If your users have a password, then we recommend hashing it using the ``bcrypt``
-encoder, instead of the traditional SHA-512 hashing encoder. The main advantages
-of ``bcrypt`` are the inclusion of a *salt* value to protect against rainbow
-table attacks, and its adaptive nature, which allows to make it slower to
-remain resistant to brute-force search attacks.
+If your users have a password, then we recommend hashing it using the ``auto``
+encoder, instead of the traditional SHA-512 or BCrypt hashing encoders. This configuration
```

Comment:
I would drop the `instead of` part entirely. `BCrypt` is a deprecated choice, and I don't think `SHA-512` qualifies as "traditional" anymore.